frontend/tests/e2e/features/create-fishbowl.feature

```diff
@@ -6,18 +6,15 @@ Feature: Create fishbowl
         Given a logged user
         And a created Fishbowl
         And a Fishbowl by slug
-        When navigates to "/"
-        And clicks on "Schedule a fishbowl" link
-        Then gets redirect to "/fishbowl/create"
-        And sees the create fishbowl form
-        When writes "Updated fishbowl" in input "title"
+        When navigates to "/fishbowl/create"
+        And writes "Updated fishbowl" in input "title"
```

Comment:
It is better to follow the Given When Then guidelines:

* Given: for things that are set before doing anthing: like previous set data or state (logged users, fishbowl data, etc..)
* When: for things that you do on the App: like navigation , clicks on things, form filling, etc...
* Then: for things you want to observe AFTER all your interactions.

You want your tests to look like this: GIVEN some data and some conditions, WHEN I do my stuff, THEN this should happen.

Try to now mix, and to not repeat steps.

Reply:
This applied for several tests, there were some mixing between the 3 kind of steps and some repetition of the same test (that slows downs the build).

Since those are E2E tests, the less we repeat, the better.